Source/UI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -173,17 +172,7 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
             textValueOfCellType[type.index] = nil
         }
 
-        guard !hasErrorValue else {
-            editDelegate?.didFinishEditingBillingForm(successfully: false)
-            return
-        }
-
-        let areAllFieldFulfilled = style.cells.first(where: {
-          guard let style = $0.style, style.isMandatory else { return false }
-          return textValueOfCellType[$0.index]?.isEmpty == true
-        }) == nil
-
-        editDelegate?.didFinishEditingBillingForm(successfully: areAllFieldFulfilled)
+        notifyContentChangeToDelegate()
     }
 
     private func validateTextOnEndEditing(textField: BillingFormTextField) {
```

Comment:
@ehab-al-cko please leave comments that require some response on a code comment, not on final review, so we can actually discuss.

You said
> Please add more details how to test it because it's not working with me

Could you please describe what exactly does that mean? The UI tests are passing on pipeline, so it appears to work on a 3rd party machine.

Reply:
Like steps to  run UITest. It's not working with normal scheme.

Reply:
If its not working with normal scheme, could you please try to use UITest scheme instead? UITests are normally a separate target and as such commonly they require a separate scheme to run.